can_msgs/package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package format="2">.
+<package format="3">.
```

Comment:
After the `<?xml version="1.0"?>`, this should have `<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>`. `ament_xmllint` can then use the XSD to validate the contents of this file.